### PR TITLE
feature/EDU#31 Improve runner

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes #31

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,17 +16,24 @@
   "private": true,
   "scripts": {
     "lint": "eslint src --ext .js",
+    "test": "jest --testPathPattern='/src/*'",
```
Comment (ahelmberger) [inline package.json:19]: As you created a `jest.config.js` you could maybe also put the testPathPattern there to have it in one place

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+Graceful.timeout = 5000;
+Graceful.exitOnDouble = true;
+Graceful.captureExceptions = false;
+
+Graceful.on('exit', signal => {
+  console.log(`Received ${signal} signal, exiting process ...`);
+  process.exit(0);
+});
+
+process.on('uncaughtException', err => {
```
Comment (ahelmberger) [inline src/index.js:14]: This should be already handled by `Graceful.captureExceptions`

File: src/index.js [hunk 3]
```diff
@@ -1,6 +1,21 @@
+const Graceful = require('node-graceful');
 const { backup } = require('./backup');
 const { restore } = require('./restore');
 
+Graceful.timeout = 5000;
+Graceful.exitOnDouble = true;
+Graceful.captureExceptions = false;
```
Comment (ahelmberger) [inline src/index.js:7]: As we have a lot of promises, we should maybe also add `Graceful.captureRejections = true;`

Comment (adrianaluchian) [reply src/index.js]: Thanks, done.

File: src/index.js (hunk 2 above)
Comment (adrianaluchian) [reply src/index.js]: Removed, thanks!

File: package.json (hunk 1 above)
Comment (adrianaluchian) [reply package.json]: That's true, I've defined `testMatch` in the config file.

**Review by laurentiu-ilici — APPROVED** [review]

**Pull request merged**